pkg/networkservice/chains/nsmgr/heal_test.go

```diff
@@ -465,9 +439,16 @@ func TestNSMGR_CloseHeal(t *testing.T) {
 	require.NoError(t, ctx.Err())
 }
 
+func tokenTimeout(restored bool) time.Duration {
+	if restored {
+		return time.Hour
+	}
+	return 3 * time.Second
+}
```

Comment:
This adding complexity, please use constants.

Reply:
Yep, fixed.